JS/form.js

```diff
@@ -0,0 +1,25 @@
+import { getSingleStocks, stockList, spinner } from "./results.js";
+
+const searchBtn = document.getElementById("button-addon2");
+
+class SearchForm {
+  constructor() {}
+
+  async getStocks() {
+    try {
+      stockList.innerHTML = "";
+      spinner.classList.remove("d-none");
+      const searchValue = document.getElementById("search-input").value;
```

Comment:
the search element should be created/stransfered as parameter to each instance of the class